docs/tools/extension.md

```diff
@@ -1,3 +1,9 @@
+:::danger
+VSCode Extension is a third party plugin, and is not developed nor maintained by Apache Weex.
+
+Committers of Apache Weex is not responsible for any issue of VSCode Extension.
```

Comment:
I think you can just get rid of that sentence. An individual committer is never responsible anyway (imho).

Reply:
Just deleted it, thanks 😅